src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs

```diff
@@ -1793,8 +1793,8 @@ internal List<CommandMetadata> GetRemoteCommandMetadata(out Dictionary<string, s
         private void WriteProgress(string statusDescription, int? percentComplete, int? secondsRemaining)
         {
             ProgressRecordType recordType;
-            if (secondsRemaining.HasValue && secondsRemaining.Value == 0 &&
-                percentComplete.HasValue && percentComplete.Value == 100)
+            if (secondsRemaining.GetValueOrDefault() == 0 &&
+                percentComplete.GetValueOrDefault() == 100)
```

Comment:
This changes original logic. This could work but we needed to review all call sites. So I suggest more correct formally:
```suggestion
            if (secondsRemaining.HasValue && secondsRemaining.GetValueOrDefault() == 0 &&
                percentComplete.HasValue && percentComplete.GetValueOrDefault() == 100)
```

Reply:
I see, will format this soon and change other calls in a similar manner.

Thanks for your input!